server/server.js

```diff
@@ -1,13 +1,18 @@
 const path = require('path');
 const express = require('express');
+const tree = require('./tree');
 
 const app = express();
 
-app.use(express.static(path.resolve(__dirname, '../client')));
+app.use(express.static(path.resolve(__dirname, '../client/public')));
 app.listen(process.env.npm_package_config_port);
 
-app.get('/', (req, res) => {
-  res.send('Connected');
+app.get('/api/tree', (req, res) => {
+  tree.getDirectoryContent(req.query.path)
```

Comment:
might want also to check `req.query.path` being empty and sending no content response (forgot the status code)